packages/sitecore-jss-angular/src/components/placeholder.component.ts

```diff
@@ -77,6 +110,7 @@ export class PlaceholderComponent implements OnInit, OnChanges, DoCheck, OnDestr
   @Input() renderings?: Array<ComponentRendering | HtmlElementRendering>;
   @Input() outputs: { [k: string]: (eventType: unknown) => void };
   @Input() clientOnly = false;
+  @Input() sitecore: LayoutServiceContextData;
```

Comment:
I'm not really sure if it's a right approach to pass the _sitecoreContext_, may be there is a way to inject the LayoutServiceContextData?
For now it means that we need to pass it in all the placeholders within the app. May be it can be done in scope of different task "[Editing Integration] Placeholders" - so you can remove it from here for now and comment out in [component-factory](https://github.com/Sitecore/jss/pull/1870/files#diff-dfd9615d11281b24cef25930deeb15495d7861cc15e81ac8cf3786732fed0a25R77)

Reply:
@illiakovalenko I went back and forth, but I agree let's leave the injection of context to the future task. I removed for now and I'll add a note to that task to make sure to implement resolving and passing the `editMode` to the `getPlaceholder` function.